guides/common/modules/con_overview-of-recommended-disaster-recovery-plans.adoc

```diff
@@ -0,0 +1,76 @@
+[id="overview-of-recommended-disaster-recovery-plans"]
+= Overview of recommended disaster recovery plans
+
+Choose a disaster recovery plan that best helps ensure the continuity of {Project} services in your deployment.
+
+//[IMPORTANT]
+//====
+//During recovery, you will need to ensure that the hostname of your {ProjectServer} does not change.
+//The IP address can change.
+//====
+
+Virtualizing your {ProjectServer}::
```

Comment:
```suggestion
.Virtualizing your {ProjectServer}
```
I would personally prefer an informal heading for the plan names but it isn't blocking your PR.

Reply:
I actually had the same idea at one point! :) I'm afraid it didn't really improve things. So I decided to stick with my usual principle -- separate content from presentation. What I'm documenting is a list so I'm using markup for d-lists :shrug: